blaze-server/src/test/scala/org/http4s/blaze/server/ServerTestRoutes.scala

```diff
@@ -57,13 +57,19 @@ object ServerTestRoutes extends Http4sDsl[IO] {
       // ///////////////////////////////
       (
         "GET /get HTTP/1.0\r\nConnection:close\r\n\r\n",
-        (Status.Ok, Set(length(3), textPlain, connClose), "get"),
+        (Status.Ok, Set(length(3), textPlain), "get"),
```

Comment:
All removals of `connClose` from this test are because it's the default in HTTP/1.0.  There's no need for us to reiterate that.